README.md

```diff
@@ -67,42 +67,54 @@ While you are not required to use React for your plugin, it must be written in
 Javascript and React is recommended. We support both a standard Javascript API
 and a React Hooks API.
 
-#### Create a Project with React
+#### Create a Project with Vite
+
+NOTE: Facebook's [create-react-app](https://github.com/facebook/create-react-app) breaks with React 19 (see https://github.com/facebook/create-react-app/issues/17004).
+
+You should use [vite](https://github.com/vitejs/vite) instead to create custom plugins.
```

Comment:
Edit the wording to not use `instead`. Instead of treating the change as a "we used to recommend this old way but you should use this new way instead", just make it so that the new way is "the only way we recommend".